cli/drivers/WordpressComposerInstallValetDriver.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+class WordpressComposerInstallValetDriver extends ValetDriver
+{
+    /**
+     * Determine if the driver serves the request.
+     *
+     * @param  string  $sitePath
+     * @param  string  $siteName
+     * @param  string  $uri
+     * @return void
+     */
+    public function serves($sitePath, $siteName, $uri)
+    {
+        if (file_exists($composerFile = $sitePath . '/composer.json')) {
+            if (file_exists($sitePath . '/wp-config.php')) {
+                return true;
+            }
+
+            $composerJson = json_decode(file_get_contents($composerFile));
```

Comment:
Excellent idea. Tried to avoid it by looking for wp-config.php first, but this makes way more since.

The only catch I see with that is the rare case where someone might define a custom vendor directory for composer AND put wp-config.php in a subdirectory. But probably not a common enough scenario to worry about.

Checking for wp-config.php or wordpress-core-installer should hopefully account for like 99.99% of setups.